src/journal/journald-server.c

```diff
@@ -2928,3 +2973,30 @@ int config_parse_compress(
 
         return 0;
 }
+
+int config_parse_forward_to_socket(
+                const char* unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        SocketAddress* addr = data;
+        int r;
+
```

Comment:
Please add asserts for `unit`, `filename`, `line`, and `rvalue`.

Reply:
We don't need to assert line as its just unsigned and zero is a valid line number (afaik).
However I have added assertions for unit, filename and rvalue as well.